stories/__tests__/SQFormButton.stories.test.js

```diff
@@ -0,0 +1,161 @@
+
+    expect(resetButton).toBeInTheDocument();
+    expect(resetButton).toHaveAttribute('type', 'reset');
+  });
+
+  it('should call a function when clicked', () => {
+    const onClickSpy = jest.fn();
+    render(<SQFormButton onClick={onClickSpy} />);
+
+    const submitButton = screen.getByRole('button', {name: /form submission/i});
+
+    userEvent.click(submitButton);
+
+    expect(onClickSpy).toHaveBeenCalled();
+  });
+
+  it('should show as disabled', () => {
+    render(<SQFormButton isDisabled={true} />);
+
+    const submitButton = screen.getByRole('button', {name: /form submission/i});
+
+    expect(submitButton).toBeDisabled();
+  });
+
+  it('should not fire onClick when disabled', () => {
+    const onClickSpy = jest.fn();
+    render(<SQFormButton isDisabled={true} onClick={onClickSpy} />);
+
+    const submitButton = screen.getByRole('button', {name: /form submission/i});
+
+    userEvent.click(submitButton);
+
+    expect(onClickSpy).not.toHaveBeenCalled();
+  });
+});
+
+describe('Button with Test Field Tests', () => {
+  it('should render a button and a text field', () => {
+    render(<SQFormButtonWithField />);
+
+    const testField = screen.getByLabelText(/test field/i);
+
+    expect(testField).toBeInTheDocument();
+
+    const submitButton = screen.getByRole('button', {name: /form submission/i});
+
+    expect(submitButton).toBeInTheDocument();
+    expect(submitButton).toHaveTextContent('Submit');
+  });
+
+  it('should disable when shouldRequireFieldUpdates is true', () => {
+    render(<SQFormButtonWithField shouldRequireFieldUpdates={true} />);
+
+    const submitButton = screen.getByRole('button', {name: /form submission/i});
+
+    expect(submitButton).toBeDisabled();
+  });
+
+  it('should enable when text field is filled out', () => {
+    render(<SQFormButtonWithField shouldRequireFieldUpdates={true} />);
+
+    const submitButton = screen.getByRole('button', {name: /form submission/i});
+    expect(submitButton).toBeDisabled();
+
+    const testField = screen.getByLabelText(/test field/i);
+
+    userEvent.type(testField, 'Hello');
+
+    expect(submitButton).toBeEnabled();
+  });
+
+  it('should submit form when submit button clicked', () => {
+    const onClickSpy = jest.fn();
+    //const onClickOverride = jest.
+    render(
+      <SQFormButtonWithField
+        shouldRequireFieldUpdates={true}
+        onClick={onClickSpy}
+      />
+    );
+
+    const submitButton = screen.getByRole('button', {name: /form submission/i});
+    expect(submitButton).toBeDisabled();
+
+    userEvent.click(submitButton);
+    expect(onClickSpy).not.toHaveBeenCalled();
+
+    const testField = screen.getByLabelText(/test field/i);
+
+    userEvent.type(testField, 'Hello');
+    expect(submitButton).toBeEnabled();
+
+    userEvent.click(submitButton);
+
+    expect(onClickSpy).toHaveBeenCalled();
+  });
+
+  it('should be disabled when button is type reset', () => {
```

Comment:
nitpick: same idea as the comment I left on line 76.
If this test is moved into the default suite, the title of the following test would need to be updated to be more explicit about what it's testing